sale_by_packaging/models/sale_order_line.py

```diff
@@ -39,14 +39,16 @@ def _onchange_product_packaging(self):
             }
         return super()._onchange_product_packaging()
 
-    @api.constrains("product_id", "product_packaging")
+    @api.constrains("product_id", "product_packaging", "product_packaging_qty")
     def _check_product_packaging_sell_only_by_packaging(self):
         for line in self:
-            if line.product_id.sell_only_by_packaging and not line.product_packaging:
+            if not line.product_id.sell_only_by_packaging:
+                continue
+            if not line.product_packaging or line.product_packaging_qty <= 0:
```

Comment:
As product_packaging_qty is managed with digit precision, you should use float_compare()

Reply:
Thanks for your review will change that

Reply:
Changed